binaryninja/main_plugin.cc

```diff
@@ -386,6 +386,27 @@ void AnalyzeFlowBinaryNinja(BinaryNinja::BinaryView* view,
   // post processing must happen afterwards.
   call_graph->PostProcessComments();
 
+  LOG(INFO) << "Binary Ninja specific post processing";
+  for (const auto &it : flow_graph->GetFunctions())
```

Comment:
`GetFunctions()` returns an `std::map<>`, so we can use structured bindings here:

```c++
for (const auto& [address, function] : flow_graph->GetFunctions()) { // ...
```

The `address` part in that map should correspond to the function entry point, so we can also omit the call to `function->GetEntryPoint()`.

Reply:
Addressed in latest commit.